day1/ques2.rb

```diff
@@ -0,0 +1,42 @@
+# 2. WAP for students to print their school level according to grade
+# e.g -
+# if grade is between 1-5 return elementary
+# if grade is between 6-8 return middle school
+# if grade is between 9-12 return high school
+# if grade is between otherwise return college
+
+                                                #using case
+
+puts "Tell the grade :"
+grade = gets.chomp().to_i
+
+case grade
+when (1..5)
+    puts "elementary school"
```

Comment:
Spacing and indentations can be improved for all program files. Use proper title case when ever you are printing something to output